### ISSUE TITLE
"Got unknown type" using OpenAI LLM via Langchain

### ISSUE THREAD

**Issue by julianprester** [gventuri/pandas-ai#352]
### 🐛 Describe the bug

I'm trying to run pandas-ai using OpenAI's gpt-3.5-turbo model via the langchain implementation (I know I can just use the pandas-ai OpenAI implementation directly, but for various reasons I would prefer to go the langchain route). While the langchain llm seems to be correctly instantiated, I cannot use it with pandas-ai (see error below). I'm assuming this has to do with the different Chat Message types implemented in langchain. I tracked the error message down to the code in the langchain library [here](https://github.com/hwchase17/langchain/blob/1dd4236177c3e21c26773d1948759129c59520e7/langchain/chat_models/openai.py#L135).

My simple example:

```python
import os
import pandas as pd
from pandasai import PandasAI

os.environ["OPENAI_API_KEY"] = 'xxx'

df = pd.DataFrame({
    "country": ["United States", "United Kingdom", "France", "Germany", "Italy", "Spain", "Canada", "Australia", "Japan", "China"],
    "gdp": [19294482071552, 2891615567872, 2411255037952, 3435817336832, 1745433788416, 1181205135360, 1607402389504, 1490967855104, 4380756541440, 14631844184064],
    "happiness_index": [6.94, 7.16, 6.66, 7.07, 6.38, 6.4, 7.23, 7.22, 5.87, 5.12]
})

from langchain.chat_models import ChatOpenAI
llm = ChatOpenAI()
pandas_ai = PandasAI(llm=llm, verbose=True)

response = pandas_ai(df, prompt='What is the average GDP?')
print(response)
```

Error:
```
2023-07-11 20:15:45 [INFO] Question: What is the average GDP?
2023-07-11 20:15:45 [INFO] Running PandasAI with langchain_openai-chat LLM...
2023-07-11 20:15:45 [INFO] Prompt ID: 78f61bd4-0c92-4ba6-8e38-8b083a00963e
Got unknown type

Unfortunately, I was not able to answer your question, because of the following error:

Got unknown type


```

**Comment by gventuri** [comment]
@julianprester thanks a lot for reporting, it has been fixed with #354.
The fix is available in version 0.6.9!

**Comment by entzyeung** [comment]
I got this problem today suddenly. It was fine yesterday.
```
File [~/miniforge3/envs/langchain/lib/python3.9/site-packages/langchain_core/language_models/chat_models.py:690](https://file+.vscode-resource.vscode-cdn.net/Users/lorentzyeung/Documents/PycharmProjects/108_NLP/%2B%2B%2Bdatacamp/~/miniforge3/envs/langchain/lib/python3.9/site-packages/langchain_core/language_models/chat_models.py:690), in BaseChatModel.__call__(self, messages, stop, callbacks, **kwargs)
    682 @deprecated("0.1.7", alternative="invoke", removal="0.2.0")
    683 def __call__(
    684     self,
   (...)
    688     **kwargs: Any,
    689 ) -> BaseMessage:
--> 690     generation = self.generate(
    691         [messages], stop=stop, callbacks=callbacks, **kwargs
    692     ).generations[0][0]
    693     if isinstance(generation, ChatGeneration):
    694         return generation.message
...
--> 145     raise TypeError(f"Got unknown type {message}")
    146 if "name" in message.additional_kwargs:
    147     message_dict["name"] = message.additional_kwargs["name"]

TypeError: Got unknown type w
```
My langchain (0.1.1)
and openai (1.8.0)
